data/GalaxyStellarMassAlphatoIron/conversion/convertRomeroGomez2023.py

```diff
@@ -0,0 +1,95 @@
+from velociraptor.observations.objects import ObservationalData
+
+import unyt
+import numpy as np
+import os
+import sys
+
+# Exec the master cosmology file passed as first argument
+with open(sys.argv[1], "r") as handle:
+    exec(handle.read())
+
+# Cosmology
+h_sim = cosmology.h
+
+output_directory = "../"
+
+if not os.path.exists(output_directory):
+    os.mkdir(output_directory)
+
+# Read the data
+raw = np.arange(8, 12.25, 0.25)
+M_star = 10 ** raw * unyt.Solar_Mass
+Z_star = 0.04 * raw - 0.28
+error_Z_star = 0.01 * raw + 0.09
+
+# Correction factor due to the difference in (X_O/X_Fe)_Sun
+# From Grevesse & Sauval (1998) to Asplund+ (2009)
+
+O_H_Grevesse98 = 8.83
+Mg_H_Grevesse98 = 7.58
+Fe_H_Grevesse98 = 7.50
+
+O_Fe_Grevesse98 = O_H_Grevesse98 - Fe_H_Grevesse98
+Mg_Fe_Grevesse98 = Mg_H_Grevesse98 - Fe_H_Grevesse98
+
+O_over_H_Asplund09 = 8.69
+Fe_over_H_Asplund09 = 7.50
+Mg_over_H_Asplund09 = 7.6
+
+O_over_Fe_Asplund09 = O_over_H_Asplund09 - Fe_over_H_Asplund09
+Mg_over_Fe_Asplund09 = Mg_over_H_Asplund09 - Fe_over_H_Asplund09
+
+element_list = np.array(["MgFe", "OFe"])
+
+for element in element_list:
+
+    output_filename = "RomeroGomez2023_Data_" + element + ".hdf5"
```

Comment:
```suggestion
    output_filename = f"RomeroGomez2023_Data_{element}.hdf5"
```

Also, saying just `Data` in the name of the file sounds a bit vague to me, especially because there is another file that you call `RomeroGomez2023_SphDwarfsLG_Data`.

Maybe add `ATLAS3D` to the name of the file, or something similar?

Reply:
Done.